nilearn/plotting/surf_plotting.py

```diff
@@ -67,10 +549,23 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
     view : {'lateral', 'medial', 'dorsal', 'ventral', 'anterior', 'posterior'}, optional
         View of the surface that is rendered. Default='lateral'.
 
-    cmap : matplotlib colormap, str or colormap object, optional
-        To use for plotting of the stat_map. Either a string
-        which is a name of a matplotlib colormap, or a matplotlib
-        colormap object. If None, matplotlib default will be chosen.
+    engine : {'matplotlib', 'plotly'}, optional
+
+        .. versionadded:: 0.8.1
+
+        Selects which plotting engine will be used by plot_surf.
+        Currently, only matplotlib and plotly are supported.
+
+        .. note::
+            To use 'plotly' and save figures to disk you should
+            have both `plotly` and `kaleido` installed.
+
+        Default='matplotlib'.
+
+    cmap : matplotlib/plotly colormap, str or colormap object, optional
+        To use for plotting of the stat_map. Either a string which is a
+        name of a colormap, or a colormap object.
+        If None, matplotlib/plotly default will be chosen.
```

Comment:
```suggestion
        If None, matplotlib/plotly default will be chosen.
``` matplotlib for bw compatibility ?